src/Marten/Events/IsArchivedMember.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Marten.Events.Archiving;
+using Marten.Linq;
+using Marten.Linq.Members;
+using Marten.Linq.Parsing.Operators;
+using Weasel.Postgresql;
+using Weasel.Postgresql.SqlGeneration;
+
+namespace Marten.Events;
+
+internal class IsArchivedMember: IQueryableMember, IComparableMember, IBooleanMember
+{
+    private static readonly string _locator = "d.is_archived";
```

Comment:
Should this be a `const` instead?

Reply:
Does it matter?

Reply:
A little bit, as the JIT doesn't need to go through Tier-0 -> Tier-1 for this and then allocate it on the frozen heap.
So for startup time (if that matters) the `const` is cheaper.

And for consistency with other places where `const` is used.